app/qml/form/MMFormPage.qml

```diff
@@ -281,7 +281,11 @@ Page {
         property var fieldWidget: model.EditorWidget
         property var fieldConfig: model.EditorWidgetConfig
 
-        property bool fieldIsReadOnly: root.state === "readOnly" || !AttributeEditable
+        // Following issue #3457, error message can be displayed even for non-editable attributes
+        // Thus, we split attributes indicating edit mode for formPage and attribute editability
+        property bool fieldIsReadOnly: root.state === "readOnly"
+        property bool isAttributeEditable: AttributeEditable
```

Comment:
Let's keep the naming convention here - all properties injected into this loader start with `fieldXYZ`

```suggestion
        property bool fieldIsEditable: AttributeEditable // Field is not editable when "Apply default value on update" is set
```